ui/dashboards/src/components/Variables/VariableEditor.tsx

```diff
@@ -180,23 +180,25 @@ export function VariableEditor(props: {
   return (
     <>
       {currentEditingVariableDefinition && (
-        <VariableEditorForm
-          initialVariableDefinition={currentEditingVariableDefinition}
-          initialAction={variableFormAction}
-          isDraft={true}
-          onSave={(definition: VariableDefinition) => {
-            setVariableDefinitions((draft) => {
-              draft[variableEditIdx] = definition;
+        <ValidationProvider>
```

Comment:
Since you include the provider here, I guess you should do the same on DatasourceEditor side?

Reply:
Yes, fixed 😄